samples/templates/default-azuredeploy.json

```diff
@@ -264,6 +274,58 @@
                 ]
             }
         },
+        {
+            "condition": "[equals(parameters('solutionType'), 'FhirServerSqlServer')]",
+            "name": "[variables('serviceName')]",
+            "type": "Microsoft.Sql/servers",
+            "apiVersion": "2015-05-01-preview",
+            "location": "[resourceGroup().location]",
+            "tags": {
+                "FhirServerSolution": "[parameters('solutionType')]"
+            },
+            "properties": {
+                "administratorLogin": "[uniqueString(resourceGroup().id)]",
+                "administratorLoginPassword": "[parameters('sqlAdminPassword')]",
+                "version": "12.0"
+            },
+            "resources": [
+                {
+                    "condition": "[equals(parameters('solutionType'), 'FhirServerSqlServer')]",
```

Comment:
Do we need these additional conditions since the parent resource already has the same condition?

Reply:
I'll have to double check.

Reply:
It fails when you do a cosmosdb deployment with the following exception when you don't include it:

```json
{
  "error": {
    "code": "ParentResourceNotFound",
    "message": "Can not perform requested operation on nested resource. Parent resource 'msh-fhir-pr-456' not found."
  }
}
```